get-digest.sh

```diff
@@ -1,38 +1,25 @@
 #!/bin/bash
 
-if [ "$#" -ne 2 ]
+if [ "$#" -ne 3 ]
 then
-    echo "Need ARO_HCP_IMAGE_ACR and REPOSITORY parameters"
+    echo "Usage: $0 <ARO_HCP_IMAGE_ACR> <REPOSITORY> <TAG>"
     exit 1
 fi
 
 aro_hcp_image_acr=${1}
 repository=${2}
+tag=${3}
 
-if [ -n "${IMAGE_DIGEST_OVERRIDE}" ];
-then
-    echo ${IMAGE_DIGEST_OVERRIDE}
-    exit 0
-fi
-
-if [ -n "${IMAGE_DIGEST}" ];
-then
-    echo ${IMAGE_DIGEST}
-    exit 0
-fi
-
-
-tags=$(mktemp)
-trap "rm ${tags}" EXIT
-
-az acr repository show-tags --orderby time_desc --n ${aro_hcp_image_acr} --repository ${repository} --detail > $tags
+tags=$(az acr repository show-tags --orderby time_desc --n "${aro_hcp_image_acr}" --repository "${repository}" --detail)
```

Comment:
I would expect a "current" deploy to be paired with the actual build of the image, so instead of looking it up from a central ACR and hoping you win the race for "latest push" could we not just record what we built and pushed, when we did it?